doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -606,6 +620,8 @@ A typical modular chassis includes a midplane-interface to interconnect the Supe
 * By default smartswitch midplane IP address assignment will be done using internal DHCP.
 * Please refer to the [ip-address-assignment document](https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md) for IP address assignment between the switch host and the DPUs.
 * The second option is the static IP address assignment.
+* A midplane state change handler will be implemented to monitor PCIe link state change events and DPU control-plane and data-plane state transitions mainly for HA.
```

Comment:
This should be a DPU state change handler that should monitor midplane and data plane states.

Reply:
Done